### PR TITLE
Use bgimage for hompage banner

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
Updated CSS to use `gbimage` plugin for homepage banner to utilize the image processing and rendering the background image in WebP format in Mobile view.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
This should improve the site performance in Mobile view by a good margin!

**Comment by l5io** [comment]
🚀 Preview for commit 94abfc57979aca08a9108e1216088ef9fb48aad4 at: https://62f23af25ec5e32a5b8f7ba9--layer5.netlify.app

File: src/sections/Home/Banner-4/index.js [hunk 1]
```diff
@@ -7,45 +7,70 @@ import Button from "../../../reusecore/Button";
...
+    graphql`
+      query {
+        backgroundImage123: file(
+          relativePath: { eq: "poster.png" }
+        ) {
+          childImageSharp {
+            gatsbyImageData(
+              width: 2000
+              quality: 50
+              webpOptions: { quality: 70 }
```
Comment (leecalcote) [inline src/sections/Home/Banner-4/index.js:30]: Whoa. Fancy. 

Teach me, oh, great and wise, MeshMate Nikhil.

**Review by leecalcote — APPROVED** [review]
👏

File: src/sections/Home/Banner-4/index.js (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/sections/Home/Banner-4/index.js:30]: Haha, sure.
Every week on Monday at 5:30 PM IST. 😂

**Pull request merged**